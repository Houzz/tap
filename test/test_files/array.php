<?php

class A {}

$a = array();
$a[1] = 10;
$a["str"] = 100;
$num = 1;
$a[$num] = 10;
$b = new A();
$a[$b] = 10;    // wrong, only string and int can be used as array key
$b[1] = 10;     // wrong, $b is not an array
$a[] = "str";

