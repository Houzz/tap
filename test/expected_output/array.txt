11 E_EXPECTED_TYPE
12 E_EXPECTED_TYPE
